src/main/scala/firrtl/passes/RemoveCHIRRTL.scala

```diff
@@ -120,8 +120,6 @@ object RemoveCHIRRTL extends Transform {
           clks += "clk"
           ens += "en"
           masks ++= create_exps(SubField(portRef, "wmask", createMask(sx.tpe)))
-          renames.rename(sx.name, s"${sx.mem}.${sx.name}.rdata")
-          renames.rename(sx.name, s"${sx.mem}.${sx.name}.wdata")
```

Comment:
Why did you remove these lines?

Reply:
um don't remember why. I'll remove these. They're probably from some other thing I was working on.